### PR TITLE
Remove getSingleProperties()

### PR DESCRIPTION
**Pull request by Henni**
This PR remove the need of `getSingleProperties()`.
also fixes https://github.com/owncloud/contacts/pull/87#issuecomment-191299862

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
rebasing ...

**Comment by Henni** [comment]
I'll do the rebase. There is a lot of broken stuff because of the PRs we just merged.

**Comment by DeepDiver1975** [comment]
after delete the fields are not yet removed from the view ...

**Comment by DeepDiver1975** [comment]
> I'll do the rebase. There is a lot of broken stuff because of the PRs we just merged.

oh - sorry @Henni - I did not see your comment

File: templates/contactDetails.html [hunk 1]
```diff
@@ -13,7 +13,9 @@
 	<button ng-click="ctrl.deleteContact()" class="icon-delete-white" title="Delete"></button>
 </header>
 <section>
-	<detailsItem ng-repeat="prop in ctrl.singleProperties" name="prop.name" data="prop.data" model="ctrl" index="prop.index" ng-class="[ 'details-item-' + prop.name ]"></detailsItem>
+	<div ng-repeat="(propName, prop) in ctrl.contact.props">
```
Comment (DeepDiver1975) [inline templates/contactDetails.html:16]: @Henni as far as I can tell this nested repeat is responsible for not updating the ui after deleting a field ...

**Comment by Henni** [comment]
`delete this.props[name][index];` doesn't work cleanly with angular.
https://github.com/owncloud/contacts/blob/remove-getSingleProp/js/models/contact_model.js#L87

**Comment by Henni** [comment]
deleting fields should work now

**Comment by DeepDiver1975** [comment]
works :+1:

**Pull request merged**